src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,50 @@
-package core.basesyntax.impl;
-
-import core.basesyntax.Storage;
-
-public class StorageImpl<K, V> implements Storage<K, V> {
-    @Override
-    public void put(K key, V value) {
-    }
-
-    @Override
-    public V get(K key) {
-        return null;
-    }
-
-    @Override
-    public int size() {
-        return -1;
-    }
-}
+package core.basesyntax.impl;
+
+import core.basesyntax.Storage;
+
+public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_LENGTH = 10;
+    private K [] keys;
+    private V [] values;
+    private int size = 0;

```

Comment:
```suggestion
    private int size;
```
0 - default value for int